app/src/components/common/Tutorial/Tutorial.tsx

```diff
@@ -28,15 +24,7 @@ export const Tutorial = ({
   };
 
   return (
-    <div
-      className={classes.root}
-      style={{
-        backgroundColor: hexToRgbA(
-          activeTheme?.colors.modes[selectedMode].base_dark,
-          0.5
-        ),
-      }}
-    >
+    <div className="screen overflow-hidden position-fixed inset-0 z-modal bg-base_dark:50">
```

Comment:
This is quite a long class, which doesn't look great.
But these cases aren't all that common, and typically mean some kind of abstraction is in order.
In this case, an `Overlay` or even `HvOverlay` component.